db/migrate/20170725143941_create_api_consumers.rb

```diff
@@ -0,0 +1,12 @@
+class CreateApiConsumers < ActiveRecord::Migration
+  def change
+    create_table :api_consumers do |t|
+      t.string :name
+      t.string :email, null: false, index: true
+      t.string :provider, null: false
+      t.string :uid, null: false, index: true
```

Comment:
not sure from where the `uid` will come, but I'm guessing these should be unique, if that's the case maybe we can add a constraint to it.